packages/fxa-auth-server/test/local/routes/recovery-keys.js

```diff
@@ -612,6 +613,63 @@ describe('POST /recoveryKey/exists', () => {
     });
   });
 
+  describe('should return estimatedSyncDeviceCount=0 with no sync devices', () => {
+    beforeEach(() => {
+      const requestOptions = {
+        credentials: { uid },
+        log,
+      };
+      return setup(
+        {
+          db: {
+            recoveryData,
+            devices: [],
+          },
+        },
+        {},
+        '/recoveryKey/exists',
+        requestOptions
+      ).then((r) => (response = r));
+    });
+
+    it('returned the correct response', () => {
+      assert.equal(response.exists, true, 'exists ');
+      assert.equal(response.estimatedSyncDeviceCount, 0);
+    });
+  });
+
+  describe('should return estimatedSyncDeviceCount=1 with sync devices', () => {
```

Comment:
Ran into some issues trying to mock the services to test oauth scope, verified locally.

Reply:
Fixed the mocking, should work now